satpy/readers/fci_l2_nc.py

```
@@ -70,26 +70,36 @@ def _get_global_attributes(self):
                 ssp_lon: longitude of subsatellite point
                 sensor: name of sensor
                 platform_name: name of the platform
+            Only for AMVs product:
+                channel: channel at which the AMVs have been retrieved
+
 
         """
         attributes = {
             "filename": self.filename,
             "spacecraft_name": self.spacecraft_name,
-            "ssp_lon": self.ssp_lon,
             "sensor": self.sensor_name,
             "platform_name": self.spacecraft_name,
+            "ssp_lon": self.ssp_lon,
         }
+
+        if product_type=="amv":
+            attributes["channel"] = self.filename_info["channel"]
+
         return attributes
 
-    def _set_attributes(self, variable, dataset_info, segmented=False):
+    def _set_attributes(self, variable, dataset_info, product_type="pixel"):
         """Set dataset attributes."""
-        if segmented:
-            xdim, ydim = "number_of_FoR_cols", "number_of_FoR_rows"
-        else:
-            xdim, ydim = "number_of_columns", "number_of_rows"
+        if product_type in ["pixel", "segmented"]:
```

Is a product of another type valid? if not should we raise an exception?

---

well yes the last case is `product_type="amv"` as it is neither pixel or segmented.